Refit/RequestBuilderImplementation.cs

```diff
@@ -788,7 +788,7 @@ IEnumerable<string> ParseEnumerableQueryParameterValue(IEnumerable paramValues,
                 using var resp = await client.SendAsync(rq, ct).ConfigureAwait(false);
                 if (!resp.IsSuccessStatusCode)
                 {
-                    throw await ApiException.Create(rq, restMethod.HttpMethod, resp, settings).ConfigureAwait(false);
+                    throw await restMethod.ErrorHandler.HandleErrorAsync(rq, restMethod.HttpMethod, resp, settings).ConfigureAwait(false);
```

Comment:
Same, what happens if the error handler returns null?

Reply:
this will now throw a NullReferenceException
it can be redone with
```diff
- throw await restMethod.ErrorHandler.HandleErrorAsync(rq, restMethod.HttpMethod, resp, settings).ConfigureAwait(false);
+ var exception = await restMethod.ErrorHandler.HandleErrorAsync(rq, restMethod.HttpMethod, resp, settings).ConfigureAwait(false);
+ if (exception != null)
+ 
+    throw exception;
+ }
```
and go according to the scenario described above